src/main/java/uk/gov/gchq/magmacore/service/DbChangeSet.java

```diff
@@ -0,0 +1,73 @@
+/*
```

Comment:
They're not transactions they're transformations, and they're in this package because they need to be exported from the module to client cod. They depend on MagmaCoreService rather than any database code (except indirectly).

Reply:
Is the `database` package not going to be exported by this module? I would've thought that's something client code might want to make use of directly rather than through the `MagmaCoreService`. Nevertheless, could they be added to `service.transformation` instead?